lib/metacat.properties.metadata.xml

```diff
@@ -395,13 +395,25 @@
         <group>4</group>
         <index>13</index>
         <description>
-            Semicolon-separated list of paths to certificate files, containing public keys to be
+            &#91;Semicolon-separated list of paths to certificate files, containing public keys to be
             used by d1_portal to verify incoming request jwt tokens (in addition to verifying
-            against the CN server).
+            against the CN server).&#93; &lt;span class="new-badge"&gt;New&lt;/span&gt;
         </description>
         <helpFile>docs/metacat-properties.html#cn-server-publiccert-filename</helpFile>
     </config>
 
+    <config>
+        <key>dataone.nodeToken.file</key>
+        <label>Admin Token Path</label>
+        <group>4</group>
+        <index>14</index>
+        <description>
+            &#91;The path to the admin jwt token that will be used in dataone-indexer to access the
+            private objects' system metadata.&#93; &lt;span class="new-badge"&gt;New&lt;/span&gt;
+        </description>
+        <helpFile>docs/metacat-properties.html/#admin-token</helpFile>
```

Comment:
please add a description to `docs/user/metacat/source/metacat-properties.rst` so this link works - see [my recent example here](https://github.com/NCEAS/metacat/pull/1813/files)

Reply:
Apologies for missing this @artntek - I have updated the `docs/user/metacat/source/metacat-properties.rst` with a new section for `dataone.nodeToken.file` along with the following description for it below - may I ask for your feedback regarding the text?
- Quick note: There is a missing period  `.` to end the last sentence, and have added this locally already. I will wait for your further feedback (to potentially include) before pushing this typo fix.

```The path to a file that contains a (JWT) token. This token will be used in the dataone-indexer to authorize access to private objects' system metadata. A valid token consists of 3 parts delimited with a period. The first section contains the header (and info about how the JWT was signed), the second  includes information on the subject (ex. id, issuer, expiry date, etc.), and the last section is the signature. ```

Reply:
Thanks, @doulikecookiedough! I would suggest making the description less technical, because admins/operators do not need to know about the internals of the token, since we will issue it for them. (correct me if I'm wrong, @taojing2002)

How about:

> The path to a file that contains an authentication token. This token will be used by the dataone-indexer, to enable indexing of private datasets.

Reply:
Sounds good to me. I've pushed the update and am going to close this PR.